srcpkgs/ccl/template

```diff
@@ -1,26 +1,31 @@
+# Template file for 'ccl'
 pkgname=ccl
-version=1.11.5
-revision=2
-wrksrc="ccl"
+version=1.12
+revision=1
 archs="i686* x86_64*"
-build_style=gnu-makefile
-hostmakedepends="m4"
+hostmakedepends="m4 tar wget"
 short_desc="Clozure Common Lisp interpreter and compiler"
-maintainer="Ankur Kothari <ankz.kothari@gmail.com>"
+maintainer="rc-05 <rc23@email.it>"
 license="Apache-2.0"
 homepage="http://ccl.clozure.com/"
-distfiles="https://github.com/Clozure/ccl/releases/download/v${version}/${pkgname}-${version}-linuxx86.tar.gz"
-checksum=b80850d8d6ca8662499975f1cd76bf51affdd29e2025796ddcff6576fe704143
+distfiles="https://github.com/Clozure/ccl/archive/v${version}.tar.gz"
+checksum=774a06b4fb6dc4b51dfb26da8e1cc809c605e7706c12180805d1be6f2885bd52
 nopie=1
 nostrip=1
 disable_parallel_build=1
+python_version=3
 
 case $XBPS_MACHINE in
 	x86_64*) _arch=64 ;;
 	i686*) broken="Build hangs forever" ;;
 	*) _arch='' ;;
 esac
 
+pre_build() {
+	wget https://github.com/Clozure/ccl/releases/download/v${version}/linuxx86.tar.gz
```

Comment:
Add this to distfiles instead.

Reply:
Added but it can't find the extracted files.

Reply:
You need to move it into proper place, see srcpkgs/cgit/template .